lib/tessel/name.js

```diff
@@ -102,6 +102,11 @@ Tessel.prototype.rename = function(opts, callback) {
   else {
     self.getName(function(err, oldName) {
       if (err) return callback && callback(err);
+      // Don't set the new name if it's the same as the old name
+      if (oldName == opts.newName) {
+        tessel.logs.info("Name of device is already", oldName);
```

Comment:
How about you print it as `tessel.logs.warn`? I think there should be some effort to bring it to the user's attention that they're doing something wrong.